.github/workflows/podvm_publish.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright Confidential Containers Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build and push the pod VM images.
+---
+name: Publish pod VM Images
+on:
+  # Create release Pod VM Images
+  release:
+    types: [created]
```

Comment:
This is fine for now, but I wonder if longer term we want to have a master release workflow that calls this via `workflow_call` so we can do things like trigger the CAA build and release tagging at the same time?

Reply:
hi @stevenhorsman ,

By "release tagging" you meant tagging the image (e.g `quay.io/confidential-containers/cloud-api-adaptor:v0.9.0` and `quay.io/confidential-containers/podvm-generic-ubuntu-amd64:v0.9.0`)?

I was thinking in send a follow up PR to tag the podvm image, yes. I wasn't thinking about the CAA image, it seems a good idea btw